dowhy/causal_refuters/bootstrap_refuter.py

```diff
@@ -1,33 +1,99 @@
 from dowhy.causal_refuter import CausalRefuter, CausalRefutation
 import numpy as np
+import random
 from sklearn.utils import resample
 import logging
 
 class BootstrapRefuter(CausalRefuter):
     """
-    Refute an estimate by running it on a random sample of the original data.
+    Refute an estimate by running it on a random sample of the data containing measurement error in the 
+    confounders. This allows us to find the ability of the estimator to find the effect of the 
+    treatment on the outcome.
+    
     It supports additional parameters that can be specified in the refute_estimate() method.
-    - 'num_simulations': int, CausalRefuter.DEFAULT_NUM_SIMULATIONS by default
-    The number of bootstrap simulations to be run
+    
+    Parameters
+    -----------
+    -'num_simulations': int, CausalRefuter.DEFAULT_NUM_SIMULATIONS by default
+    The number of simulations to be run
     - 'sample_size': int, Size of the original data by default
     The size of each bootstrap sample
+    - 'required_variables': int, list
+    A user can input either an integer value or a list.
+        1. An integer argument refers to how many confounders  will be modified
+        2. A list allows the user to explicitly refer to which confounders should be seleted to be made noisy
+    - 'noise': float, BootstrapRefuter.DEFAULT_STD_DEV by default
+    The standard deviation of the noise to be added to the data
+    - 'probability_of_change': float, 'noise' by default

```

Comment:
Better to specify here that if not given, the probabiliy_of_change will be assumed to be the noise variable (if less than 1)